
image_save_path: src/livox_hikcamera_cal/aruco_img/
image_load_path: src/livox_hikcamera_cal/aruco_img/
image_format: png

yaml_save_path: cfg/

dictionary_name: 11
selected_ids: [12, 150, 364, 544, 712]
use_center_id: false
center_id: 12
aruco_marker_size: 500

aruco_real_length: [252, 252, 252, 252, 252]

caliboard_width: 1004
caliboard_height: 804

# criteria_iter_times: 1000000
# iter_difference: 0.00000001

# show_corners_image: false
